<h3 style="margin:0px">Class: org.apache.etch.examples.chat.TestRemoteChatServer (7 methods) </h3><br><pre style="padding:0"><code class="html"><div id="tags"><button id="5"style="border-width:0;padding:0;background:#282B2E"  onclick="filter('5')" data-toggle="tooltip" title="Executes methods or other tests from the same test unit"><kbd id="tag-5"class="label-info"style="display: inline-block;font-size:7pt" >ExecutionTester&nbsp;(5)</kbd></button>&nbsp;<button id="16"style="border-width:0;padding:0;background:#282B2E"  onclick="filter('16')" data-toggle="tooltip" title="Releases resources used by the test cases"><kbd id="tag-16"class="label-info"style="display: inline-block;font-size:7pt" >TestCleaner&nbsp;(2)</kbd></button>&nbsp;</div></code></pre><center><button onclick="showBlocks()" style="margin:6px">Update filter <span class="glyphicon glyphicon-refresh" aria-hidden="true"></span> </button></center><br>
<pre class="type-5 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
/** 
 * testLogin2: User is already logged in
 * @throws Exception
 */
@Test(expected=Failure.class) public void testLogin2() throws Exception {
  Log.report("TestRemoteChatServer.login: testLogin2 - Login user. User is already logged in");
  cs=setupAndConnectUser(user);
  cs.login(user,user);
  cs.login(user,user);
}

</code></pre>

<pre class="type-5 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
/** 
 * testLogin4: Login - bad password
 * @throws Exception
 */
@Test(expected=Failure.class) public void testLogin4() throws Exception {
  Log.report("TestRemoteChatServer.login: testLogin4 - Login user. Bad password");
  cs=setupAndConnectUser(user);
  cs.login(user,badPass);
}

</code></pre>

<pre class="type-16 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Releases resources used by the test cases">TestCleaner</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Releases resources used by the test cases
"></span><br>
/** 
 * Logout out user and close TCP conection
 * @throws Exception 
 */
@After public void cleanup() throws Exception {
  if (cs != null) {
    cs.logout();
    cs._stop();
  }
}

</code></pre>

<pre class="type-16 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Releases resources used by the test cases">TestCleaner</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Releases resources used by the test cases
"></span><br>
/** 
 * @throws Exception
 */
@AfterClass public static void stopRemoteChatListener() throws Exception {
  if (listener != null) {
    listener.transportControl(Transport.STOP_AND_WAIT_DOWN,4000);
    listener=null;
  }
}

</code></pre>

<pre class="type-5 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
/** 
 * testLogin1: Login user
 * @throws Exception
 */
@Test public void testLogin1() throws Exception {
  Log.report("TestRemoteChatServer.login: testLogin1 - Login user");
  cs=setupAndConnectUser(user);
  cs.login(user,user);
}

</code></pre>

<pre class="type-5 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
/** 
 * testLogin3: Login - user is null
 * @throws Exception
 */
@Test(expected=Failure.class) public void testLogin3() throws Exception {
  Log.report("TestRemoteChatServer.login: testLogin3 - Login user. User is null");
  cs=setupAndConnectUser(nullUser);
  cs.login(nullUser,nullUser);
}

</code></pre>

<pre class="type-5 "><code><span class="label label-info" style="display: inline-block;" data-toggle="tooltip" title="Executes methods or other tests from the same test unit">ExecutionTester</span>&nbsp;<span class=" glyphicon glyphicon-comment" aria-hidden="true" label-info" data-toggle="tooltip" title="This method/test case: 
- Executes methods or other tests from the same test unit
"></span><br>
/** 
 * testLogin5: Login - bad username
 * @throws Exception
 */
@Test(expected=Failure.class) public void testLogin5() throws Exception {
  Log.report("TestRemoteChatServer.login: testLogin5 - Login user. Bad username");
  cs=setupAndConnectUser(user);
  cs.login(user,badUser);
}

</code></pre>

<script>$(document).ready(function() {
  $('pre code').each(function(i, block) {
    hljs.highlightBlock(block);
  });
});</script>
